src/extensionConsole/pages/deployments/DeploymentsContext.tsx

```
@@ -50,30 +48,21 @@ import {
 } from "@/utils/extensionUtils";
 import useAutoDeploy from "@/extensionConsole/pages/deployments/useAutoDeploy";
 import { activateDeployments } from "@/extensionConsole/pages/deployments/activateDeployments";
+import { useGetDeploymentsQuery } from "@/data/service/api";
+import { deserializeError } from "serialize-error";
+import { isEqual } from "lodash";
+import useMemoCompare from "@/hooks/useMemoCompare";
+
+function getError(deploymentsError: unknown, permissionsError: unknown) {
+  if (deploymentsError) {
+    if (deserializeError(deploymentsError).name === "ExtensionNotLinkedError") {
```

Potential suggestion - wonder if we should leave a comment/give the result of this boolean a helpful name to explain why we're ignoring this error